app/assets/javascripts/components/account_page.es6.jsx

```diff
@@ -0,0 +1,54 @@
+class AccountPage extends Component {
+
+  static get defaultState() {
+    return { sidebar: true };
+  }
+
+  get styles() {
+    return {
+      container: {
+        display: 'flex',
+        flexDirection: 'column',
+        position: 'relative',
+        width: '100vw',
+        height: '100vh',
+      },
+      body: {
+        flex: 1,
+        position: 'relative',
+        display: 'flex', 
+        flexDirection: 'row',
+  			justifyContent: 'center',
```

Comment:
Spacing here for some reason is funny (there might be a hidden tab on the left side).